.travis.yml

```diff
@@ -35,6 +34,7 @@ env:
   global:
     - "S3_BUCKET=fixmystreet-bundle-cache"
     - "CYPRESS=0"
+    - "CATALYST_NOWARN_DEPRECATE=1"
```

Comment:
could we add a comment to say why this is here. Also add might be nice to add it to `scripts/test`.